src/components/product-gallery/ProductGalleryUI.tsx

```diff
@@ -59,24 +59,25 @@ export const ProductGalleryUI = ({
 
     <div style={{ display: "flex" }}>
       <SidePanel value={storeSelected} setValue={setStore} />
-      <Content
-        isFetching={isFetching}
-        searchTerm={debouncedSearchTerm}
-        emptyResults={items.length === 0}
-        ContentEmptyComponent={ProductGalleryContentEmpty}
-        ContentNoResultComponent={ProductGalleryContentNoResult}
-        width="80%"
-      >
-        {/* TODO: use a list view in place of this icons view */}
-        <ContentList
-          items={items}
-          checkedItemIds={checkedItemIds}
-          toggleCheckedItem={toggleCheckedItem}
-          selectItem={selectItem}
-          hasNextPage={hasNextPage}
-          fetchNextPage={fetchNextPage}
-        />
-      </Content>
+      <div style={{ width: "80%", height: "700px" }}>
```

Comment:
I would add a TODO here, something like: `// TODO: replace the hardcoded styles once the rules have been defined on the styles sheet`.